### DISCUSSION TITLE
Deleting mail in another account

### DISCUSSION THREAD

**Discussion by santos-joj-cano** [community/community#116329] [GitHub Education]
### Select Topic Area

General

### Body

I currently want to be able to access GitHub educational experiences, but I had an issue with my email where it is linked to another account and I need to delete the account with the educational email I had in order to use it in the new account I have. Anyone else with the same problem?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Owl-in-night , we're happy you're here! You are more likely to get a useful response if you are posting your question  in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hi @Owl-in-night  :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.
